Dockerfile

```diff
@@ -1,13 +1,13 @@
-FROM dockerfile/java:oracle-java7
+FROM dockerfile/java:oracle-java8
```

Comment:
Is java8 now required for new versions of Mirth? I looked here: http://www.mirthcorp.com/community/wiki/display/mirth/System+Requirements but the page is out of date.